source/includes/_projects.md

```diff
@@ -1,11 +1,176 @@
 # Projects
 
+## Viewing projects
+
+```shell
+curl https://api.handshq.com/v1/projects \
+  -H "Accept: application/json" \
+  -H "Authorization: bearer [api_token]"
+```
+
+This endpoint allows you to view projects for the company who is registered with the api key you provide.
+
+### HTTP Request
+
+`GET https://api.handshq.com/v1/projects`
+
+### Response
+
+Successful requests will return a json payload of that division's projects and a `200` status code.
+
+> 200
+
+```json
+  {
+    "data": [
+      {
+        "id": "1234",
+        "type": "project",
+        "attributes": {
+          "name": "Test Project",
+          "reference": "ABC1",
+          "start_date": "2021-12-20",
+          "end_date": "2022-12-21",
+          "archived_at": null,
+          "state": "not_submitted"
+        },
+        "relationships": {
+          "fields": {
+            "data": [
+              {
+                "id": "123",
+                "type": "field"
+              },
+              {
+                "id": "234",
+                "type": "field"
+              }
+            ]
+          },
+          "user": {
+            "data": {
+              "id": "345",
+              "type": "user"
+            }
+          }
+        },
+        "links": {
+          "app_url": "https://app.handshq.com/projects/1234"
+        }
+      },
+      {
+        "id": "5678",
+        "type": "project",
+        "attributes": {
+          "name": "Test Project 2",
+          "reference": "ABC2",
+          "start_date": "2021-12-20",
+          "end_date": "2022-12-21",
+          "archived_at": null,
+          "state": "not_submitted"
+        },
+        "relationships": {
+          "fields": {
+            "data": [
+              {
+                "id": "123",
+                "type": "field"
+              },
+              {
+                "id": "234",
+                "type": "field"
+              }
+            ]
+          },
+          "user": {
+            "data": {
+              "id": "345",
+              "type": "user"
+            }
+          }
+        },
+        "links": {
+          "app_url": "https://app.handshq.com/projects/5678"
+        }
+      }
+    ]
+  }
+```
+
+## Viewing projects by project reference
+
+```shell
+curl https://api.handshq.com/v1/projects?reference=#{project_reference} \
```

Comment:
Should we be using square bracket notation for where user value is required? (number of places)

`curl https://api.handshq.com/v1/projects?reference=[project_reference] \`